src/DatePicker/__stories__/DatePicker.stories.js

```diff
@@ -87,8 +87,14 @@ readOnly.storyName = 'ReadOnly';
 
 export const localized = () => (
     <div className='fddocs-container'>
-        <DatePicker locale='es' />
-        <DatePicker locale='fr' />
+        <DatePicker
+            dateFormat='DD/MM/YYYY'
+            defaultValue='20/07/2020'
+            locale='es' />
+        <DatePicker
+            dateFormat='DD/MM/YYYY'
+            defaultValue='20/06/2020'
+            locale='fr' />
```

Comment:
I can revert the changes to the storybook examples, but this was the situation where the bug was reported